cijoe/templates/perf_report/cover.jinja2.tmpl

```diff
@@ -0,0 +1,48 @@
+
+.. image:: xnvme.png
+   :align: center
+   :width: 100%
+
+.. raw:: pdf
+
+   Spacer 0 10mm
+
+.. class:: title
+
+`xNVMe`_ in `SPDK`_
+
+.. class:: subtitle
+
+Performance Report
+
+.. class:: body
+
+Performed the:
+
+* INSERT_DATE
+
+by the **xNVMe Automation Infrastructure** which consists of
+
+* `CIJOE in xNVMe`_ / `CIJOE`_ 
+* `GitHUB Actions in xNVMe`_ / `GHA`_
+
+with **acknowledgements** to:
+
+* Simon A. F. Lund
+* Karl Bonde Torp
+* Jim Harris
+
+For questions to, and discussion of, the `xNVMe`_ in `SPDK`_ performance
+report, please find the acknowledged parties in the **xNVMe** discord
+and **SPDK** Slack channels.
```

Comment:
Maybe we should link to these?

Reply:
Right, added to improvements on https://github.com/OpenMPDK/xNVMe/issues/371